<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutDown</class>
 <widget class="QMainWindow" name="ShutDown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShutDown</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>30</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入关机的时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_hour">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>小时：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hour">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_minute">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分钟：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_minute">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
